src/components/CreateProductForm.jsx

```diff
@@ -1,7 +1,108 @@
-import React from "react";
+import React from 'react';
+import { Formik, Field, Form, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import toast, { Toaster } from 'react-hot-toast';
 
-const CreateProductForm = () => {
-  return <div>CreateProductForm</div>;
+const CreateProductForm = ({ onSubmit }) => {
+  const validationSchema = Yup.object({
+    title: Yup.string().required('Title is required'),
+    price: Yup.number().required('Price is required').positive('Price must be positive'),
+    description: Yup.string().required('Description is required'),
+    category: Yup.string().required('Category is required'),
+    images: Yup.mixed().required('Image is required'),
+  });
+
+  const initialValues = {
+    title: '',
+    price: '',
+    description: '',
+    category: '',
+    images: null,
+  };
+
+  const handleSubmit = async (values, { setSubmitting }) => {
```

Comment:
Can you tell me please is there need to use async/awit in this function and try/catch we do not use an API call here ?